code/segmentedtickerplant/stplog.q

```diff
@@ -299,6 +299,7 @@ init:{[dbname]
 
 // Close logs on clean exit
 .z.exit:{
+  .proc.initialised:1b;
```

Comment:
Can you add an explanatory comment above this line? It may not be obvious why this is necessary when someone else looks at this function